INTITA/Modules/Welcome/WelcomeViewController.swift

```diff
@@ -135,12 +123,16 @@ class WelcomeViewController: UIViewController, Storyboarded, UIScrollViewDelegat
         startBtn.rounded()
     }
     func setupSkipBtn() {
-        skipBtn.titleLabel?.adjustsFontSizeToFitWidth = true
-        skipBtn.setTitle("skip".localized, for: .normal)
-        skipBtn.titleLabel?.font = UIFont.primaryFontLight
-        skipBtn.titleLabel?.textAlignment = .left
-        
-        arrowImg.image = UIImage(named:"skipArrow")
+        let skipStackView = UIStackView(frame: .zero)
+        let imgView = UIImageView(frame: CGRect(x: 150, y: 0, width: 10, height: 24))
+        imgView.image = UIImage(named:"skipArrow")!
```

Comment:
Try to avoid force unwrapping. Not a big thing at this moment might be an issue in future.

Reply:
done